src/app/project-overview/component.ts

```diff
@@ -200,6 +213,32 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
       .subscribe();
   }
 
+  private _loadClusterMachineDeployments(cluster: Cluster): void {
+    if (Health.allHealthy(this.clusterHealth[cluster.id]) && !cluster.deletionTimestamp) {
+      this._machineDeploymentService
+        .list(cluster.id, this.project.id)
+        .pipe(takeUntil(this._unsubscribe))
+        .subscribe({
+          next: machineDeployments => {
+            const mdStatus: MachineDeploymentStatus = {
+              replicas: 0,
+              availableReplicas: 0,
+              updatedReplicas: 0,
+            } as MachineDeploymentStatus;
+            machineDeployments.forEach(machineDeployment => {
+              mdStatus.replicas += machineDeployment.status?.replicas || 0;
+              mdStatus.availableReplicas += machineDeployment.status?.availableReplicas || 0;
+              mdStatus.updatedReplicas += machineDeployment.status?.updatedReplicas || 0;
+            });
+            this.clusterMDStatus = {
```

Comment:
IMO, we can keep same as of List page? or other way around to change it? wdyt?

```suggestion
    this.clusterMDStatus[cluster.id] = mdStatus;
```